README.md

```diff
@@ -150,30 +161,26 @@ You can also build p4-constraint in a Docker container, for example:
 ```sh
 docker build --tag p4-constraints .                 # Time to get coffee...
 docker run --tty --interactive p4-constraints bash  # Open shell in container.
-bazel test //p4_constraints/...
+bazel test //...                                    # Run tests in container.
 ```
 
 ## Golden tests
 
 The easiest way to experiment with p4-constraints is to write a
 [golden test](https://ro-che.info/articles/2017-12-04-golden-tests).
-We provide [Bazel rules](e2e-test/p4check.bzl) `run_p4check` and `diff_test` to make
-this convenient.
+We provide [Bazel rules](e2e-test/p4check.bzl) `run_p4check` and `diff_test` to
+make this convenient.
 See the [e2e-test/](e2e-test/) folder -- in particular
 [e2e-test/BUILD.bazel](e2e-test/BUILD.bazel) -- for examples of how to use them.
 
-Currently, the golden tests require [p4c](https://github.com/p4lang/p4c) as
-a system dependency (we hope to make this a source dependency in the future):
-```sh
-which p4c  # This needs to succeed for golden tests to work.
-```
-You may want to use our [Docker container](#docker), which comes with p4c
-preinstalled.
-
 To run all golden tests, execute
 ```sh
 bazel test //e2e-test/...
 ```
+[Recall](building) that this will build p4c and requires
```

Comment:
Does this link work? What does it link to? (not sure how to test this myself on github).

Reply:
It links to the `## Building` section of the README. (To test, click ... at the top of the README.md diff and then "View File".)

Reply:
And sure enough, the link was broken. It needed a '#'.
Good catch, thanks!